.github/workflows/release.yml

```diff
@@ -148,10 +148,10 @@ jobs:
       - name: Release
         run: |
           cd gradle-plugin
-          ./gradlew bintrayUpload
-        env:
-          BINTRAY_USER: ${{ secrets.BINTRAY_USER }}
-          BINTRAY_KEY: ${{ secrets.BINTRAY_KEY }}
-
+          TAG_NAME=${GITHUB_REF#refs/tags/}
```

Comment:
From the [docs](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables):

> The branch or tag ref that triggered the workflow. For example, `refs/heads/feature-branch-1`. If neither a branch or tag is available for the event type, the variable will not exist.

Nice.

Reply:
Worth pointing out that this configuration assumes the job will be triggered always and only from a tag.

That is, if we'll need to remember to update this "before we start publishing the plugin for each PR commit and commits on `trunk`". 

You're likely already aware of this 😄 But I called it out to add it to our shared awareness.

Reply:
Yeap I was aware of it and briefly indicated this in the PR description, but it's still good for you to raise it as you pointed out for shared awareness :)